engine/apps/slack/scenarios/alertgroup_appearance.py

```diff
@@ -45,148 +39,17 @@ def process_scenario(self, slack_user_identity, slack_team_identity, payload):
             "message_ts": message_ts,
         }
 
-        integration = alert_group.channel.integration
-
-        PAYLOAD_TEXT_SIZE = 3000
-        raw_request_data = json.dumps(alert_group.alerts.first().raw_request_data, sort_keys=True, indent=4)
-
-        # This is a special case for amazon sns notifications in str format CHEKED
-        if (
-            hasattr(AlertReceiveChannel, "INTEGRATION_AMAZON_SNS")
-            and alert_group.channel.integration == AlertReceiveChannel.INTEGRATION_AMAZON_SNS
-            and raw_request_data == "{}"
-        ):
-            raw_request_data = alert_group.alerts.first().message
-
-        raw_request_data_chunks = [
-            raw_request_data[i : i + PAYLOAD_TEXT_SIZE] for i in range(0, len(raw_request_data), PAYLOAD_TEXT_SIZE)
-        ]
-        for idx, chunk in enumerate(raw_request_data_chunks):
-            block = {
-                "type": "input",
-                "block_id": f"payload_{idx}",
-                "label": {
-                    "type": "plain_text",
-                    "text": f"Payload (Part {idx + 1}):" if len(raw_request_data_chunks) > 1 else "Payload (Readonly)",
-                },
-                "element": {
-                    "type": "plain_text_input",
-                    "placeholder": {
-                        "type": "plain_text",
-                        "text": "Payload of the current alert",
-                    },
-                    "action_id": UpdateAppearanceStep.routing_uid(),
-                    "multiline": True,
+        alert_receive_channel = alert_group.channel
+        blocks = [
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": f":point_right: Click <{alert_receive_channel.web_link}|here> to open Integrations settings, edit Slack templates and return here",
```

Comment:
nit: Maybe use a button with a URL as shown in [Slack docs](https://api.slack.com/reference/block-kit/block-elements#button) to redirect to the web UI? A button with URL seems to be more appropriate for this use case

Reply:
In the next version we’ll have 3 urls to 3 templates